src/utils/CustomDeleteDialog.jsx

```diff
@@ -0,0 +1,71 @@
+import { useA11yDialog } from 'react-a11y-dialog';
+import { createPortal } from 'react-dom';
+
+const CustomDeleteDialog = ({ text, deleteConfirmation }) => {
+	// `instance` is the `a11y-dialog` instance.
+	// `attr` is an object with the following keys:
+	// - `container`: the dialog container
+	// - `overlay`: the dialog overlay (sometimes called backdrop)
+	// - `dialog`: the actual dialog box
+	// - `title`: the dialog mandatory title
+	// - `closeButton`:  the dialog close button
+	const [instance, attr] = useA11yDialog({
+		id: 'confirm-delete-dialog',
+		role: 'dialog',
+		title: 'Confirm delete dialog',
+	});
+
+	const dialog = createPortal(
+		<div {...attr.container} className="dialog-container">
+			<div {...attr.overlay} className="dialog-overlay" />
+
+			<div {...attr.dialog} className="dialog-content">
+				<h1 {...attr.title} className="dialog-title">
+					Confirm Delete Dialog
+				</h1>
+
+				<p>{text}</p>
+
+				<button {...attr.closeButton} className="dialog-close">
+					<img
+						src="../../public/img/x.png"
```

Comment:
I'm not sure if this was just an issue I'm experiencing locally, but the close button a
![Screen Shot 2022-08-25 at 4 23 16 PM](https://user-images.githubusercontent.com/90880701/186786466-c3afa569-e60e-4711-80b7-2a8f5e4cee01.png)
ppears to be a broken image when I tried testing...

Reply:
Following up this. I realized I was just using the preview link and was not running the code locally. Once I did, the close button appears to be working as intended! I did want to mention though, it might be helpful to add in your testing steps that if you have not already done so, you will need to run `npm install --save react-a11y-dialog` to install the library, otherwise, the app will not render.

Reply:
^^ Thank you, @Trevor-Rezac!! Was wondering why it's been load/spinning for a few mins...

Reply:
I always forget about the dependencies, thank you, I'll add it to the instructions